lib/log.go

```diff
@@ -27,7 +27,7 @@ func (m LogMethods) Name() string {
 // Attributes defines attributes for each method
 func (m LogMethods) Attributes() map[string]AttributeSet {
 	return map[string]AttributeSet{
-		"history":        {AEHistory, "POST"},
+		"history":        {AELog, "POST"},
 		"entries":        {AEEntries, "POST"},
```

Comment:
To be clear here, I just made this name change from `/logbook` to `/entries` for clarify, but always intended it to be temporary until we adjust to the new spec. When we do our final refactor, `/entries` should be `/log` and `/history` should be removed!

Reply:
Awesome, and yeah I'll be keeping rebasing this as I'm trailing all the other changes.